CHANGES.md

```diff
@@ -104,6 +104,10 @@
   + Handle let operator punning uniformly with other punning forms.
     Normalizes let operator to the punned form where possible, if output syntax version is at least OCaml 4.13.0. (#1834, #1846, @jberdine)
 
+  + Added optionnal arguments to the format command to set or override the config and path. (#1935, @panglesd)
+
+  + Added a command to ask the server to take the configuration given by the ocamlformat’s dot files for a specific path. (#1935, @panglesd)
```

Comment:
Maybe we should add a separate section for the RPC (both server and client) in the changelog

Reply:
I added a `RPC changelog` section